ESLint add-on missing `@types/node`

```plaintext
┌  Welcome to the Svelte CLI! (v0.9.4)
│
◇  Which template would you like?
│  SvelteKit minimal
│
◇  Add type checking with TypeScript?
│  Yes, using TypeScript syntax
│
◆  Project created
│
◇  What would you like to add to your project? (use arrow keys / space bar)
│  prettier, eslint
```

```js
// eslint.config.js

// Cannot find module 'node:url' or its corresponding type declarations.ts(2307)
import { fileURLToPath } from 'node:url';
```

This can be fixed by

```shell
pnpm add @types/node -D
```

Reference

- https://github.com/sveltejs/cli/issues/509

--- a/.changeset/hungry-waves-like.md
+++ b/.changeset/hungry-waves-like.md
@@ -1,0 +1,5 @@
+---
+'sv': patch
+---
+
+fix(eslint): add `@types/node` in devDependencies

--- a/packages/addons/eslint/index.ts
+++ b/packages/addons/eslint/index.ts
@@ -10,7 +10,7 @@ import {
 	type AstTypes
 } from '@sveltejs/cli-core/js';
 import { parseJson, parseScript } from '@sveltejs/cli-core/parsers';
-import { addEslintConfigPrettier } from '../common.ts';
+import { addEslintConfigPrettier, getNodeTypesVersion } from '../common.ts';
 
 export default defineAddon({
 	id: 'eslint',
@@ -25,6 +25,7 @@ export default defineAddon({
 		sv.devDependency('eslint-plugin-svelte', '^3.0.0');
 		sv.devDependency('globals', '^16.0.0');
 		sv.devDependency('@eslint/js', '^9.18.0');
+		sv.devDependency('@types/node', getNodeTypesVersion());
 
 		if (typescript) sv.devDependency('typescript-eslint', '^8.20.0');
 

